index.html

```diff
@@ -759,796 +678,43 @@ <h1>Common Constraints</h1>
+      that are common in their geographic region are not globally applicable 
+      and may lead to misinterpretation with drastic consequences.
+      </p>
+      <p><span class="rfc2119-assertion" id="common-constraints-units">
+        It is highly RECOMMENDED to provide a <code>unit</code>, 
+        if a value has a physical quantity.</span>
+      </p>
+      <p><span class="rfc2119-assertion" id="common-constraints-units-metric">
+        It is highly RECOMMENDED to use the metric system (SI units) 
+        for devices that are used in global deployments.</span>
+      </p>
+    </section>
+    <!--  Date Format -->
 
-      <!-- Forms -->
-      <section id="forms">
-        <h3>Forms</h3>
-        <p>
-        </p>
-        <section>
-          <h4>Mandatory fields</h4>
-          <div class="rfc2119-assertion" id="profile-thing-forms-mandatory-fields-1">
-            <p>
-              The following fields MUST be present in a form
-              element of a <em>Baseline TD</em>:
-            </p>
-            <table class="def">
-              <thead>
-                <tr>
-                  <th>keyword</th>
-                  <th>type</th>
-                  <th>constraints</th>
-                </tr>
-              </thead>
-              <tbody>
-              </tbody>
-            </table>
-          </div>
-        </section>
-        <section>
-          <h4>Additional Constraints</h4>
-          <div class="rfc2119-assertion" id="profile-5-1-7-2-thing-forms-additional-constraints-1-x">
-            <p>
-              The following additional constraints MUST be
-              applied to the Form elements of a <a>WoT Thing
-                Description</a> conforming to the <a>Baseline
-                profile</a>:
-            </p>
-            <table class="def">
-              <thead>
-                <tr>
-                  <th>keyword</th>
-                  <th>type</th>
-                  <th>constraint</th>
-                </tr>
-              </thead>
-              <tbody>
-              </tbody>
-            </table>
-          </div>
-        </section>
-      </section>
+    <section id="common-constraints-date-format">
+      <h2 id="date-format">Date format</h2>
+      <p>
+        <span class="rfc2119-assertion" id="common-constraints-date-format-1">
+          All date and time values MUST use the canonical dateTime representation format
+          defined in section 3.2.7 of [xmlschema-2].</span>
+      </p>
+      <p>
+        As described by this section the following constraints must be observed:
+        <span class="rfc2119-assertion" id="common-constraints-date-format-2">
+          All dateTime values SHOULD use UTC as the time zone and use the 'Z' identifier.</span>
+      </p>
+      <p>
+        <span class="rfc2119-assertion" id="common-constraints-date-format-3">
+          A time value of 24:00 is not permitted; the value 00:00 MUST be used instead.</span>
+      </p>
```

Comment:
In specref, [[xmlschema-2]] points to https://www.w3.org/TR/xmlschema-2/ which doesn't appear to have a section 3.2.7.

That spec does say _"'24' is permitted if the minutes and seconds represented are zero"_, which is not what you want.

In https://github.com/w3c/wot-profile/pull/267 I proposed using the reference [[RFC3339]] (https://www.rfc-editor.org/rfc/rfc3339) instead, since that already says:

_"Although ISO 8601 permits the hour to be "24", this profile of ISO 8601 only allows values between "00" and "23" for the hour in order to reduce confusion."_

RFC 3339 also only supports UTC and only numerical offsets (not those that don't have an integral number of minutes). I believe this resolves the ambiguity of timezones you are concerned about, without being quite so prescriptive.

In short, if you just reference RFC3339, the rest of the qualifying text is not needed.

Reply:
This is text we previously had agreed on. If it needs to be improved, let's treat it in a separate issue.
I filed a new issue for a follow up.